notebooks/summit/AuxTel/SITCOM-426/SITCOM-426_PSFandPhotometry_Validation.ipynb

```diff
@@ -0,0 +1,918 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0735e70c-d746-4165-a131-327d1709b8f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script.log.level = logging.DEBUG"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d1edef79-5812-4f67-9ef8-dd9d3f1a405a",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Write start info into EFD"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4b7a35e4-1564-43fa-9edc-554f394664c3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script.log.info(f'START- SITCOM-426 PSF&Photometry Validation -- at {Time.now()}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ec420c4-c86c-4068-a4a7-58fbbef41103",
+   "metadata": {},
+   "source": [
+    "## Assess that conditions meet criteria- sub-arcsec seeing and photometric\n",
+    "Photometric conditions clear definition?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "434431e1-96e3-44f4-8667-38cb0b2f039d",
+   "metadata": {},
+   "source": [
+    "## CWFS\n",
+    "A CWFS is recommended just before the test to ensure the system is in focus and the parabolic focus sweep is symmetrical."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c26aff69-a5ba-414f-ae90-5b7afff2ae3b",
+   "metadata": {},
+   "source": [
+    "### Slew to target"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be80f78c-8ff9-4cee-ade3-cb6b46f92616",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### Declare target "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77bea4ae-f3b1-4671-b127-390e2bd435d1",
+   "metadata": {},
+   "source": [
+    "Query for a target with a magnitud limit of Vmag of 8 and around the same area of the sky as the target source."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6cfbf353-52a0-4de9-97ee-c22289f0bc92",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "target = await script.atcs.find_target(az=180.2, el=75, mag_limit=8)\n",
+    "logger.info(f'Target for CWFS with mag_lim 8 is {target}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa2a9f54-82b1-44df-af4b-380044c702f6",
+   "metadata": {},
+   "source": [
+    "#### Slew to the defined target"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f3f6d290-1645-4836-93a0-bcba0fab4ef9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await script.atcs.slew_object(target, rot_type=RotType.PhysicalSky)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d2319e9-9be1-4c08-9930-5e457f29ae55",
+   "metadata": {},
+   "source": [
+    "Uncomment the following line to take a snapshot to verify the target is positioned as expected "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bd4c8b32-d03c-48b2-a2cf-61df3b9ad6b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# exposure = await latiss.take_engtest(\n",
```

Comment:
I know this is commented out, but, when taking acquisition images, please use `latiss.take_acq`, so the images have the correct header info.